configure.d/1_bio_clone.conf

```diff
@@ -40,6 +43,12 @@ apply() {
             {
                 return bio_clone_fast(bio, gfp_mask, NULL);
             }" ;;
+    "4")
+        add_function "static inline struct bio *cas_bio_clone(struct bio *bio, gfp_t gfp_mask)
```

Comment:
is "cas_bio_clone" used anywhere? I don't see any reference in my sources.

Reply:
removed altogether